guidelines/project-best-practices.md

```diff
@@ -143,7 +143,7 @@ The [Project Incubation Exit Criteria](../governing-documents/project-incubation
 * Document Long-term support (LTS) release strategy - [Fabric example](https://github.com/hyperledger/fabric-rfcs/blob/main/text/0005-lts-release-strategy.md)
```

Comment:
Do we know which projects will remain in github.com/hyperledger versus move to github.com/LF-Decentralized-Trust?

Reply:
Only Besu moving, no changes needed here...